Marlin/src/inc/SanityCheck.h

```diff
@@ -1698,7 +1698,7 @@ static_assert(hbm[Z_AXIS] >= 0, "HOMING_BUMP_MM.Z must be greater than or equal
 /**
  * Pins and Sensor IDs must be set for each heater
  */
-#if HEATER_0_USES_MAX6675 && !PIN_EXISTS(MAX6675_SS)
+#if HEATER_0_USES_MAX6675 && !PIN_EXISTS(MAX6675_SS) && !(PIN_EXISTS(MAX31855_CS) || PIN_EXISTS(MAX31865_CS) || PIN_EXISTS(MAX6675_CS))
```

Comment:
Again, multiple names for that same thing: CS and SS. We should choose and use only one, in my option.

Reply:
Since @thinkyhead has already done work on this area, I defer to his expertise.

Reply:
I fixed this so that MAX????_SS_PIN is now put into MAX????_CS_PIN in conditionals_post.h file. When SanityCheck gets executed the code will use the MAX????_CS_PIN variables.